trace_replay/trace_replay.cc

```diff
@@ -325,4 +395,85 @@ Status Replayer::ReadTrace(Trace* trace) {
   return TracerHelper::DecodeTrace(encoded_trace, trace);
 }
 
+void Replayer::BGWorkGet(void* arg) {
+  ReplayerWorkerArg ra = *(reinterpret_cast<ReplayerWorkerArg*>(arg));
+  delete reinterpret_cast<ReplayerWorkerArg*>(arg);
```

Comment:
Instead of making a copy of ReplayerWorkerArg and delete the original one, can we just put it into a unique pointer?